src/ExposureHistory/History/index.tsx

```diff
@@ -39,7 +40,10 @@ const History: FunctionComponent<HistoryProps> = ({
     navigation.navigate(Screens.MoreInfo)
   }
 
-  const showExposureHistory = exposures.length > 0
+  const recentExposures = exposures.filter((e) =>
```

Comment:
Oh I didn't realize that! I thought that since we were switching from a 21-day view to a 14-day view, we would need to filter down. Is there any concern about calling it a 14 day view if exposures will disappear after a week?

Reply:
oops, meant to say older thank 14 days.  so we expect to only ever receive exposures from 0 to 14 days ago.  This of course is coming over an api that is actively being worked on so, might make sense to have a belt and suspenders type approach.  Personally leaning towards relying on the api as much as possible and reducing the complexity of our app as much as possible as it makes more strategic sense for the OS to be handling the EN data.